Web_Development/FrontEnd/CSS Margin/CSS Margin.md

```diff
@@ -0,0 +1,85 @@
+# CSS Margin
+It can be used for creating space outside the element by this we can adjust the position of element like where we have to shift it.
+There are properties for setting the margin for each side of an element (top, right, bottom, and left).

```

Comment:
The sentence is not clear & understandable. Rephrase it. Also, don't use words like "you" "we".